From 96fb993615f7a35e515b58e3bb79392a484f8b94 Mon Sep 17 00:00:00 2001
From: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
Date: Thu, 22 Jul 2021 08:27:29 +0200
Subject: [PATCH 36/98] ARM: dts: meson8: Add the PWM_C (DV9) and PWM_D pins

There are some Meson8m2 boards which don't use a PMIC (like Ricoh
RN5T618) but use two PWM regulators for VCCK and VDDEE. Add the PWM_C
(DV9) and PWM_D pins so the pinctrl settings can be applied on those
boards.

Signed-off-by: Martin Blumenstingl <martin.blumenstingl@googlemail.com>

[dts: rebased onto 6.6.144 for 6.6-lts - i2c_b_pins doesn't exist yet at
 this baseline (added upstream after 6.6.144), so inserted the two new pin
 groups after pwm_e_pins instead; same pinmux content]
---
diff --git a/arch/arm/boot/dts/amlogic/meson8.dtsi b/arch/arm/boot/dts/amlogic/meson8.dtsi
index cd52813ec..c194dd285 100644
--- a/arch/arm/boot/dts/amlogic/meson8.dtsi
+++ b/arch/arm/boot/dts/amlogic/meson8.dtsi
@@ -560,6 +560,22 @@ mux {
 			};
 		};
 
+		pwm_c_dv9_pins: pwm-c-dv9 {
+			mux {
+				groups = "pwm_c_dv9";
+				function = "pwm_c";
+				bias-disable;
+			};
+		};
+
+		pwm_d_pins: pwm-d {
+			mux {
+				groups = "pwm_d";
+				function = "pwm_d";
+				bias-disable;
+			};
+		};
+
 		uart_a1_pins: uart-a1 {
 			mux {
 				groups = "uart_tx_a1",
